akka-projection-cassandra/src/test/scala/akka/projection/cassandra/ContainerSessionProvider.scala

```diff
@@ -29,3 +29,10 @@ final class ContainerSessionProvider extends CqlSessionProvider {
       .build()
   }
 }
+
+object ContainerSessionProvider {
+  val Config =
+    """
+      |akka.projection.cassandra.session-provider = "akka.projection.cassandra.ContainerSessionProvider"
+      |""".stripMargin
```

Comment:
nitpick: no need for margin, and stripMargin

Reply:
:+1: